.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Ist die nötig oder kann man sie auch in die normale .gitignore einfügen?

Reply:
Ich bin ja generell kein Fan davon überhaupt den .idea/ Ordner im git aufzunehmen, aber ohne ich kann man das Projekt nicht so einfach in CLion öffnen. Die .gitignore wird aber von IntelliJ gesteuert und kann daher ruhig dort bleiben